src/main/java/mate/academy/spring/exceptions/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,43 @@
+package mate.academy.spring.exceptions;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.validation.ObjectError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@ControllerAdvice
+public class GlobalExceptionHandler extends ResponseEntityExceptionHandler {
+    @Override
+    protected ResponseEntity<Object> handleMethodArgumentNotValid(MethodArgumentNotValidException
+                                                                              ex,
+                                                                  HttpHeaders headers,
+                                                                  HttpStatus status,
+                                                                  WebRequest request) {
+        Map<String, Object> body = new LinkedHashMap<>();
```

Comment:
```suggestion
    protected ResponseEntity<Object> handleMethodArgumentNotValid(
    	MethodArgumentNotValidException ex,
    	HttpHeaders headers,
    	HttpStatus status,
    	WebRequest request) {
        Map<String, Object> body = new LinkedHashMap<>();
```
можна трохи зсунути вліво всі параметри